src/algokit_utils/application_specification.py

```diff
@@ -62,7 +63,7 @@ class DefaultArgumentDict(TypedDict):
     data: int | str | bytes | MethodDict
 
 
-StateDict = TypedDict(  # noqa: UP013  # can't convert as "global" is a reserved keyword
+StateDict = TypedDict(  # can't convert as "global" is a reserved keyword
```

Comment:
oh where did the `noqa` go? did ruff get a smarts boost?

Reply:
Yeah latest version said it wasn't required anymore and removed it

Reply:
Just occurred to me that comment that is left is not needed now

Reply:
Removed comment